veggieseasons/README.md

```diff
@@ -1,42 +1,10 @@
 # Veggie Seasons
 
-An iOS app that shows which fruits and vegetables are currently in season. It
-showcases Flutter's Cupertino package.
-
-[Available now in the App Store!](https://itunes.apple.com/is/app/veggie-seasons/id1450855435)
-
-**NOTE:** While Flutter supports many platforms, this application is designed
-specifically for iOS. It's not intended to be run on Android, web, or desktop.
-
-## Goals
-
-* Show how to build an interface that iOS users will feel right at home
-  with.
-* Show how Flutter's Cupertino widgets work together.
-
-## The important bits
-
-### `/screens/*`
-
-These are the screens presented in the app, roughly analogous to
-UIViewControllers. `HomeScreen` is the root, and the others are shown
-as the user navigates.
-
-### `/widgets/search_bar.dart`
-
-An example of how to construct an Cupertino-style search bar. The
-Flutter team [is working on an official widget](https://github.com/flutter/flutter/issues/9784)
-for this. Once that effort is complete, developers will not need to roll
-their own search bars, so to speak.
-
-## Questions/issues
-
-If you have a general question about any of the techniques you see in
-the sample, the best places to go are:
-
-* [The FlutterDev Google Group](https://groups.google.com/forum/#!forum/flutter-dev)
-* [The Flutter Gitter channel](https://gitter.im/flutter/flutter)
-* [StackOverflow](https://stackoverflow.com/questions/tagged/flutter)
-
-If you run into an issue with the sample itself, please file an issue
-in the [main Flutter repo](https://github.com/flutter/flutter/issues).
+The Veggie Seasons app has temporarily been moved to the
+[experimental](https://github.com/flutter/samples/tree/master/experimental/veggieseasons)
+directory of this repository due to the integration of state restoration into
+the app. Not all features of state restoration are available on the stable
+channel of flutter yet. Once commit
+[flutter/flutter@053ebf2](https://github.com/flutter/flutter/commit/053ebf2c080c7b8efbf4020683a5ba27d9daa3b8)
+has reached the stable channel, the app can move back to its old home in
+this directory.
```

Comment:
Consider throwing something like

    For more info on how to run the samples in the `experimental` directory,
    see the [README](experimental/README.md) in that folder.

onto the end, just in case anyone's confuse as to why it went to that folder specifically.